### DISCUSSION TITLE
Unable to Submit Feedback

### DISCUSSION THREAD

**Discussion by ghost** [community/community#69209] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When I or Someone submit their **feedback**(made with html form) it says 405 error.....?

**Comment by queenofcorgis** [comment]
Hi @compknow , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by jjrh92** [comment]
The 405 Method Not Allowed error occurs when the web server is configured in a way that does not allow you to perform a specific action for a particular URL. It's an HTTP response status code that indicates that the request method is known by the server but is not supported by the target resource. Check [https://youtu.be/yPx43AbsPBU](https://youtu.be/yPx43AbsPBU)
